Competition2017/src/competition/subsystems/shooter_wheel/commands/RunShooterWheelsForRangeCommand.java

```diff
@@ -0,0 +1,26 @@
+package competition.subsystems.shooter_wheel.commands;
+
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem.ShooterRange;
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+
+public class RunShooterWheelsForRangeCommand extends BaseShooterWheelCommand {
+
+    ShooterWheelsManagerSubsystem wheelsManager;
+    ShooterRange range;
+    
+    public RunShooterWheelsForRangeCommand(ShooterRange range, ShooterWheelSubsystem shooterWheelSubsystem) {
```

Comment:
We'll need this concept at some point to be able to map buttons to these distances. I agree the capacity for dynamic ranging somewhere lower down seems reasonable for this command this feels fine to me.